src/docket/tasks.py

```diff
@@ -44,7 +45,13 @@ async def fail(
     )
 
 
+async def sleep(seconds: float) -> None:
```

Comment:
nice! was gonna suggest to add this to `trace` before I saw you had this here

Reply:
Yes!  That's gonna help with the demo too